docker/README.md

```diff
@@ -0,0 +1,73 @@
+# 🪐 Docker Image and Container for Big Data Harvest Client
+
+## 🏃 Steps to build the docker image of the Big Data Harvest Client
+
+#### 1. Update (if required) the following version in the `Dockerfile` with a compatible Big Data Harvest Client version.
+
+| Variable                        | Description |
+| ------------------------------- | ------------|
+| big_data_harvest_client_version | The version of the Big Data Harvest Client release to be included in the docker image|
+
+```    
+# Set following argument with a compatible Big Data Harvest Client version
+ARG big_data_harvest_client_version=1.0.0-SNAPSHOT
+```
+
+#### 2. Open a terminal and change the current working directory to `big-data-harvest-client/docker`.
+
+#### 3. Build the docker image as follows.
+
+```
+docker image build -t nasapds/big-data-harvest-client .
+```
+
```

Comment:
I would like the docker push to be part of the readme.

Because the objective of the procedure is to publish on artifactory (https://hub.docker.com/u/nasapds) the docker image so that it can be re-used by the user without a need to build it himself.

Reply:
Thanks for the suggestion! I updated all 3 related README files with instructions to push the images.